### PR TITLE
{bio}[foss/2022a] QIIME2 v2023.7.0

### PR DESCRIPTION
**Pull request by Neves-P**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Neves-P** [comment]
Test report by @Neves-P
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
WKS058930 - Linux Ubuntu 22.04, x86_64, 12th Gen Intel(R) Core(TM) i7-1260P, Python 3.10.12
See https://gist.github.com/Neves-P/efb733df903bd52b8dc8f70b41774857 for a full test report.

File: easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+name = 'QIIME2'
+version = '2023.7.0'
+
+homepage = 'https://qiime2.org'
+description = """QIIME 2 is a powerful, extensible, and decentralized microbiome bioinformatics platform that is free,
+open source, and community developed."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+dependencies = [
```
Comment (akesandgren) [inline easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb:11]: Add blank line between toolchain and dependencies

File: easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+    }),
+    ('ijson', '3.2.3', {
+        'checksums': ['10294e9bf89cb713da05bc4790bdff616610432db561964827074898e174f917'],
+    }),
+    ('iow', '1.0.6', {
+        'modulename': 'bp',
+        'preinstallopts': """sed -i "/'nose /d" setup.py && """,
+        'checksums': ['ac4f579a0881f1c827f75033a304341ae91d3a527cc6aed3fea53749f1b9951c'],
+    }),
+    ('atpublic', '4.0', {
```
Comment (akesandgren) [inline easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb:149]: Please readd the comment from 2023.5.1 about why we're using whls for atpublic and flufl.lock

File: easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+        'checksums': ['8a4493f8bdb73fb9655dcd5115bc897e207319e36c8835f39c516a2d7e9d79a1'],
+    }),
+    ('wrapt', '1.14.1', {
+        'checksums': ['380a85cf89e0e69b7cfbe2ea9f765f004ff419f34194018a6827ac0e3edfed4d'],
+    }),
+    ('formulaic', '0.4.0', {
+        'checksums': ['087950518c53a2491deb52b8ddd66833f953882e517a42e476c007cc8f6892c5'],
+    }),
+    ('altair', '5.0.1', {
+        'preinstallopts': """sed -i -e 's/^build-backend = .*/build-backend = "setuptools.build_meta"/g' -e 's/^requires = .*/requires = ["setuptools"]/g' -e 's/^license-files = .*//g' -e 's/dynamic = \["version"\]/version = "%(version)s"/g' pyproject.toml &&  schemadir=%(installdir)s/lib/python%(pyshortver)s/site-packages/altair/vegalite/v5/schema/ &&  mkdir -p $schemadir && cp altair/vegalite/v5/schema/vega-lite-schema.json $schemadir && """,
```
Comment (akesandgren) [inline easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb:172]: This should just be the same as in the 2023.5.1 version
```
'preinstallopts': _preinstallopts,
```

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb:12]: fixed in 5496f24cd8

File: easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb:151]: fixed in 5496f24cd8

File: easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb (hunk 3 above)
Comment (boegel) [reply easybuild/easyconfigs/q/QIIME2/QIIME2-2023.7.0-foss-2022a.eb]: fixed in 5496f24cd8

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3171.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/18b7052d5d04d0ff3def8dbb28aff1ff for a full test report.

**Comment by Neves-P** [comment]
Sorry, both, I'd forgotten to address the review. Thanks for making the changes @boegel! 

I've fixed the file permissions in 128bc04

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3138.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/f2021e0c5fdf3ff8be2aabfa6ef5659a for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**